stable/airflow/README.md

```diff
@@ -254,6 +254,7 @@ The following table lists the configurable parameters of the Airflow chart and t
 | `airflow.webReplicas`                    | how many replicas for web server                        | `1`                       |
 | `airflow.config`                         | custom airflow configuration env variables              | `{}`                      |
 | `airflow.podDisruptionBudget`            | control pod disruption budget                           | `{'maxUnavailable': 1}`   |
+| `airflow.secretsMapping`		   | override any environment variable with a secret	     | 				 |
```

Comment:
Can you add a line or two on the "Use precreated secret for postgres and redis" section in the readme?

Reply:
Just reword this in the README:
```
Due to the above, created a generic solution that will map any env var in the deployment and stateful set to a given secret in an existingAirflowSecret.

Secrets can be mapped in your values.yaml like so:
airflow:
existingAirflowSecret: my-airflow-secrets
secretsMapping:

    * envVar: AIRFLOW__LDAP__BIND_PASSWORD
      secretName: ldapBindPassword

    * envVar: POSTGRES_USER
      secretName: airflowPostgresUser

    * envVar: POSTGRES_PASSWORD
      secretName: airflowPostgresPassword

    * envVar: REDIS_PASSWORD
      secretName: airflowRedisPassword
```